NOTICE-binary

```diff
@@ -52,3 +52,363 @@ PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF
 LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING 
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS 
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+---------------------------------------------------------------------
+This software includes the nvCOMP SDK with the following licenses:
+
+LICENSE AGREEMENT FOR NVIDIA SOFTWARE DEVELOPMENT KITS
+
+This license agreement("Agreement”) is a legal agreement between you and NVIDIA Corporation
+("NVIDIA") and governs your use of the NVIDIA nvCOMP software development kit as available at
+NVIDIA’s discretion (each, a “SDK”).
```

Comment:
(each, a "SDK") looks like a typo?

Reply:
I've contacted the license author and am waiting for an update.  In the meantime I'd like to merge in what we have so far.

Reply:
That's OK with me.